### PR TITLE
DDF 1150

### PR DESCRIPTION
**Pull request by ghost**
UI look and feel improvements - dropdown selectors for types and sources in Search panel.

[Review on Reviewable](https://reviewable.io/reviews/codice/ddf-ui/59)

### CODE REVIEW

**Comment by kcwire** [comment]
@mplachec the unit tests are failing on for Travis CI. I restarted them but still had no luck. Can you look into the issue?

**Comment by shaundmorris** [comment]
@pklinef this one is in your queue. Looks like the Travis build is successful, not sure if anything else was being worked on this PR.

File: search-ui/standard/src/main/webapp/templates/search/searchForm.handlebars [hunk 1]
```diff
@@ -257,7 +257,8 @@
             <div class="tab-content form-group">
                 <div id="noTypeTab" class="tab-pane {{#ifNotOr type}}active{{/ifNotOr}}"></div>
                 <div id="typeTab" class="tab-pane col-lg-10 {{#if type}}active{{/if}}">
-                    <select id="typeList" name="type" multiple="multiple">
+                    <select class="select-types" id="typeList" name="type" multiple title="Select Types" data-size="10" data-container="body" data-width="auto"
```
Comment (andrewkfiedler) [inline search-ui/standard/src/main/webapp/templates/search/searchForm.handlebars:260]: The selects on line 260 and 331 are already differentiable through their id, so you can eliminate the class on each.

File: search-ui/standard/src/main/webapp/js/view/Query.view.js [hunk 2]
```diff
@@ -410,9 +411,9 @@ define([
                 var federationSourcesSelectOptions = _.clone(multiselectOptions);
                 federationSourcesSelectOptions.noneSelectedText = 'Select Sources';
 
-                this.$('#typeList').multiselect(typeSelectOptions).multiselectfilter();
+                this.$('.select-types').selectpicker();
```
Comment (andrewkfiedler) [inline search-ui/standard/src/main/webapp/js/view/Query.view.js:414]: It's possible to consolidate lines 414 and 416.  

```
this.$('select[multiple]').selectpicker();
```

File: search-ui/standard/src/main/webapp/js/view/Query.view.js [hunk 3]
```diff
@@ -593,8 +594,8 @@ define([
                 this.model.setDefaults();
                 // Refresh the multiselect lists to reflect the changes
                 // to the model
-                $('#typeList').multiselect("refresh");
-                $('#federationSources').multiselect("refresh");
+                $('.select-types').selectpicker('refresh');
```
Comment (andrewkfiedler) [inline search-ui/standard/src/main/webapp/js/view/Query.view.js:597]: It's possible to consolidate lines 597 and 598.

```
$('select[multiple]').selectpicker('refresh');
```

File: search-ui/standard/src/main/webapp/js/view/Query.view.js [hunk 4]
```diff
@@ -366,11 +367,11 @@ define([
                 // changes when sources are added/removed or
                 // modified (e.g., become available/unavailable)
                 this.sources.bind('add change remove', function() {
-                    $('#federationSources').multiselect("refresh");
+                    $('.select-sources').selectpicker('refresh');
```
Comment (andrewkfiedler) [inline search-ui/standard/src/main/webapp/js/view/Query.view.js:370]: If you eliminate the class as suggested, you can reference these using id instead.

```
$('#federationSources').selectpicker('refresh');
```

File: search-ui/standard/src/main/webapp/js/view/Query.view.js [hunk 5]
```diff
@@ -366,11 +367,11 @@ define([
...
                 });
 
                 this.types.bind('add change remove', function() {
-                    $('#typeList').multiselect("refresh");
+                    $('.select-types').selectpicker('refresh');
```
Comment (andrew-fiedler) [inline search-ui/standard/src/main/webapp/js/view/Query.view.js]:
```
$('#typeList').selectpicker('refresh');
```

File: search-ui/standard/src/main/webapp/css/searchPage.css [hunk 6]
```diff
@@ -1579,6 +1579,19 @@ a.pause:hover, a.pause:focus {
     font-size: 12px;
 }
 
+#typeTab .bootstrap-select > .btn {
```
Comment (andrewkfiedler) [inline search-ui/standard/src/main/webapp/css/searchPage.css:1582]: You could combine the rules on lines 1582 and 1586:

```
#typeTab .bootstrap-select > .btn,
#sources .bootstrap-select > .btn {
       width: 220px;
}
```

**Comment by kcwire** [comment]
@pklinef any issues you need resolved before merging this?

**Pull request merged**